src/EditmodeContext.js

```diff
@@ -3,5 +3,5 @@ import { createContext } from "react";
 export const EditmodeContext = createContext({
   branch: null,
   projectId: null,
-  chunkFallback: null,
+  defaultChunks: null,
```

Comment:
This can default to `[]`, since `defaultChunks` is expected to be an array.

This way, your logic can stay the same, whether it's provided or not.

(`defaultChunks.find` when `defaultChunks.length === 0` is negligible)